.circleci/config.yml

```diff
@@ -38,7 +38,6 @@ workflows:
               only: /.*/
       - check-bundle-size:
           requires:
-            - prepare
```

Comment:
Note: `prepare` is upstream from both `build` and `install-mbx-ci` so it's implicitly required anyway.